### ISSUE TITLE
Generic.Files.LineLength never throw error when PSR2 is used

### ISSUE THREAD

**Issue by nmfzone** [squizlabs/PHP_CodeSniffer#2463]
### Environment
MacOSX High Sierra

PHP v7.2
PHP_CodeSniffer v3.4.1

### Steps to reproduce

1. Add `Generic.Files.LineLength` to the ruleset
    ```
    <rule ref="Generic.Files.LineLength" />
    ```
2. Add whole PSR2 to the ruleset
    ```
    <rule ref="PSR2" />
    ```

### What is expected ?

When code exceed 100 length (default) should throw an error.

### What is actually happening?

When code exceed 100 length (default) never throw an error.

### Additional Comments

When PSR2 not used, it's work as excpected.

**Comment by gsherwood** [comment]
By including the PSR2 standard, you're including the PSR2 line length rules, which are:
- a warning at 120 characters
- no errors, ever

If you don't want those settings to apply, you need to override them, like this:
```xml
<rule ref="PSR2" />
<rule ref="Generic.Files.LineLength">
    <properties>
        <property name="lineLimit" value="80"/>
        <property name="absoluteLineLimit" value="100"/>
    </properties>
</rule>
```

**Comment by nmfzone** [comment]
@gsherwood Nice one, I've already figured it out. But I'm not sure, where I can find about this in documentation.

Could you give me where is the ruleset defined by PSR2? Will close the issue asap.

**Comment by nmfzone** [comment]
Lol, already find it in PSR2 directory. Sorry about this.
